silkrpc/core/evm_trace.cpp

```diff
@@ -61,6 +61,13 @@ std::ostream& operator<<(std::ostream& out, const TraceConfig& tc) {
     return out;
 }
 
+void from_json(const nlohmann::json& json, TraceCall& cm) {
+    if (json.is_array()) {
+        cm.call = json.at(0);
+        cm.trace_config = json.at(1);
+    }
```

Comment:
If input JSON is not an array, the `TraceCall` instance will be empty. Is it a valid value? If it is valid, shouldn't we explicitly accept an empty JSON value and reject other non-array values (e.g. a number followed by `null`)?

Reply:
You're right (it was not finished yet...). 
I decided to remove the if because it is better to leave `nlohmann::json` doing check and raise exception for us. 
By the way, we need a review in all case we parse the a wrong input JSON to get parameters as, in these cases, the response form `silkrcpdaemon` differ from `rcpdaemon` ones.